package.json

```diff
@@ -28,6 +28,7 @@
     "ember-cli-deploy": "0.5.1",
     "ember-cli-deploy-lightning-pack": "0.2.0",
     "ember-cli-dotenv": "1.0.4",
+    "ember-cli-es5-shim": "^0.1.1",
```

Comment:
Needed for mirage. It uses `.bind()` internally, which PhantomJs doesn't support out of the box. This shim adds that support.